subsys/net/lib/nrf_provisioning/nrf_provisioning_cbor.patch

```diff
@@ -2,53 +2,53 @@ diff --git a/subsys/net/lib/nrf_provisioning/include/nrf_provisioning_cbor_encod
 --- a/subsys/net/lib/nrf_provisioning/include/nrf_provisioning_cbor_encode.h
 +++ b/subsys/net/lib/nrf_provisioning/include/nrf_provisioning_cbor_encode.h
 @@ -21,7 +21,7 @@
-extern "C" {
-#endif
+ extern "C" {
+ #endif
```

Comment:
??? and why is there a patch file anyway

Reply:
@nordicjm Any further comments?